### PR TITLE
Create javadoc for only openj9 source files

### PR DESCRIPTION
**Pull request by lumpfish**
Signed-off-by: Simon Rushton <rushton@uk.ibm.com>

### CODE REVIEW

**Comment by lumpfish** [comment]
Jenkins compile aix,osx,zlinux,win jdk15

File: closed/custom/Docs.gmk [hunk 1]
```diff
@@ -103,18 +100,214 @@ OPENJ9_JAVADOC_SCRIPT_BODY = \
...
+
+# If running on Windows, the source file names in the response file must be converted to
+# Windows format before passing them to javadoc.
+OPENJ9_ONLY_JAVADOC_FIXPATH_FILELIST := $(foreach file,$(OPENJ9_ONLY_JAVADOC_FILELIST),$(call FixPath,$(file)))
+
+$(OPENJ9_ONLY_JAVADOC_MARKER) : $(OPENJDK_JAVADOC_MARKER) $(OPENJ9_ONLY_FILELIST)
+	$(call LogWarn, Generating javadoc for openj9 source files)
+	$(eval $(call ListPathsSafely,OPENJ9_ONLY_JAVADOC_FIXPATH_FILELIST,$(OPENJ9_ONLY_JAVADOC_FILES)))
+	for dir in $(OPENJDK_ONLY_PKGLIST) ; do \
+ 		$(MKDIR) -p $(OPENJ9_STUBS_DIR)/$${dir} ; \
```
Comment (keithc-ca) [inline closed/custom/Docs.gmk]: Please remove the leading space (before the first tab) to match jdk16.
Then squash.

Comment (lumpfish) [reply closed/custom/Docs.gmk]: Whitespace removed and squashed.

**Review by keithc-ca — APPROVED** [review]
Good PR builds: https://ci.eclipse.org/openj9/view/Pull%20Requests/job/PullRequest-OpenJDK15/4/

**Pull request merged**